src/main.cpp

```diff
@@ -3,6 +3,6 @@
 
 int main()
 {
-    std::cout << "LogicalImages!" << std::endl;
+    std::cout << "LogicalImages!\n";
```

Comment:
This change is not meaningful ;)

Reply:
In other files we were using '\n', so to keep to one style I modified it.